src/BugsnagUnity/Client.cs

```diff
@@ -429,7 +429,17 @@ public void SetApplicationState(bool inFocus)
                     if (Configuration.AutoTrackSessions
                      && BackgroundStopwatch.Elapsed.TotalSeconds > AutoCaptureSessionThresholdSeconds)
                     {
-                        SessionTracking.StartSession();
+                        if (IsUsingFallback())
+                        {
+                            SessionTracking.StartSession();
+                        }
+                        else
+                        {
+                            if (Application.platform.Equals(RuntimePlatform.Android))
+                            {
+                                SessionTracking.ResumeSession();
+                            }
```

Comment:
Why only Android?

Reply:
Because Android cannot listen to the activity lifecycle whereas Cocoa does right?

Reply:
Ahh 💡 not obvious - maybe worthy of a comment?

Reply:
Done 👍